rust/src/lib/iface.rs

```diff
@@ -188,7 +188,7 @@ impl From<&str> for InterfaceState {
             "up" => Self::Up,
             "down" => Self::Down,
             "absent" => Self::Absent,
-            "ignore" => Self::Ignore,
+            "ignore" => Self::unknown,
             _ => Self::Unknown,
         }
     }
```

Comment:
@EjiroLaurelD can you take a look at how the `BaseInterface` struct is defined here
https://github.com/nmstate/nmstate/blob/base/rust/src/lib/ifaces/base.rs

Reply:
@ihuguet  made a very important input here:
https://github.com/nmstate/nmstate/issues/2452#issuecomment-2010133918

you can take a look at it. it should make an ultimate solution to this

Reply:
The `state: ignore` is very important for many use cases. Please add new entry instead of change it to unknown.